maestro-client/src/main/resources/maestro-driver-ios-config.xctestrun

```diff
@@ -69,7 +69,7 @@
 			<key>DYLD_FRAMEWORK_PATH</key>
 			<string>__TESTROOT__/Debug-iphonesimulator:__TESTROOT__/Debug-iphonesimulator/PackageFrameworks:__PLATFORMS__/iPhoneSimulator.platform/Developer/Library/Frameworks</string>
 			<key>DYLD_INSERT_LIBRARIES</key>
-			<string>__PLATFORMS__/iPhoneOS.platform/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libMainThreadChecker.dylib</string>
+			<string>/Library/Developer/CoreSimulator/Profiles/Runtimes/iOS 15.5.simruntime/Contents/Resources/RuntimeRoot/usr/lib/libMainThreadChecker.dylib</string>
```

Comment:
interesting why this was changed 🤔 the previous version looks more generic

Reply:
Not sure as well I checked with one of our client flows locally the artifacts work

Reply:
would it still work if you revert this change?

Reply:
I don't think so because this path was from the machine we have in script and the current one is from my local simulator. I would say lets fix this by having artifacts from staging machine